benchmarks/benchmark_serving.py

```diff
@@ -7,7 +7,7 @@
         --disable-log-requests
 
     (TGI backend)
-    ./launch_tgi_server.sh <your_model> <max_batch_total_tokens>
+    ./launch.sh --server tgi --model facebook/opt-125m --tokens 10000 -p 8000
```

Comment:
1. `tokens` is a bit confusing. Suggest to align to `--max-batch-total-
tokens`.
2. It would be better to have default values when they are not specified.